django/test/client.py

```diff
@@ -345,12 +345,16 @@ class RequestFactory:
     just as if that view had been hooked up using a URLconf.
     """
 
-    def __init__(self, *, json_encoder=DjangoJSONEncoder, **defaults):
+    def __init__(self, *, json_encoder=DjangoJSONEncoder, headers=None, **defaults):
```

Comment:
As I wrote below for `generic()`, good idea to make the new arg kwarg-only

```suggestion
    def __init__(self, *, json_encoder=DjangoJSONEncoder, *, headers=None, **defaults):
```

Reply:
Can we still make the arg keyword-only in constructors please? This comment and others still pending.

Reply:
Indeed, I missed the `[Async]Client` constructors. But the `RequestFactory.__init__` already only accepts keyword args.

Reply:
Oh yeah, this comment can be resolved.